web-client/src/presenter/computeds/trialSessionWorkingCopyHelper.caseNotes.test.js

```diff
@@ -0,0 +1,131 @@
+import { MOCK_CASE } from '../../../../shared/src/test/mockCase';
+import {
+  STATUS_TYPES,
+  TRIAL_STATUS_TYPES,
+} from '../../../../shared/src/business/entities/EntityConstants';
+import { applicationContext } from '../../applicationContext';
+import { runCompute } from 'cerebral/test';
+import { trialSessionWorkingCopyHelper as trialSessionWorkingCopyHelperComputed } from './trialSessionWorkingCopyHelper';
+import { withAppContextDecorator } from '../../withAppContext';
+describe('trial session working copy computed', () => {
+  const trialSessionWorkingCopyHelper = withAppContextDecorator(
+    trialSessionWorkingCopyHelperComputed,
+    applicationContext,
+  );
+
+  const MOCK_TRIAL_SESSION = {
+    city: 'Hartford',
+    courtReporter: 'Test Court Reporter',
+    irsCalendarAdministrator: 'Test Calendar Admin',
+    judge: { name: 'Test Judge' },
+    postalCode: '12345',
+    startDate: '2019-11-25T15:00:00.000Z',
+    startTime: '10:00',
+    state: 'CT',
+    term: 'Fall',
+    termYear: '2019',
+    trialClerk: { name: 'Test Trial Clerk' },
+    trialLocation: 'Hartford, Connecticut',
+  };
+
+  it('should set calendarNotes on cases that have them', () => {
+    const mockCaseWithCalendarNotes = '102-19';
+    const mockCaseWithoutCalendarNotes = '5000-17';
+    const mockCaseWithCalendarNotesNotInSelectedTrialSession = '999-99';
+    const mockCalendarNote = 'this is a test note';
```

Comment:
Nitpik: could these be put into a before each block?

Reply:
Good idea, moved the ones that are reused into the outer `describe` block.